src/app/app.routes.ts

```diff
@@ -0,0 +1,3 @@
+import { Routes } from '@angular/router';
```

Comment:
suggestion

          import { Routes } from '@angular/router'; // Importing Routes type from Angular router

          Explantion: It's a good practice to add a comment explaining what is being imported for better code readability.